helm/krr-enforcer/templates/enforcer-webhook.yaml

```diff
@@ -0,0 +1,33 @@
+apiVersion: admissionregistration.k8s.io/v1
+kind: MutatingWebhookConfiguration
+metadata:
+  name: {{ .Release.Name }}-krr-enforcer
+  labels:
+    app.kubernetes.io/component: krr-enforcer
+webhooks:
+  - name: krr-enforcer.{{ .Release.Name }}.{{ .Release.Namespace }}.svc
+    sideEffects: None
+    admissionReviewVersions: ["v1", "v1beta1"]
```

Comment:
Did you test both of these? If not, I'd limit to the one you tested if possible.

Reply:
fixed, removed v1veta1